graphics/text/Makefile

```diff
@@ -70,55 +70,55 @@ help.png: text_strings helpbg.png
 	   -background "helpbg.png"                  \
 	   150,5   "file:helpttl.png"                \
 	   10,25   "Weapons"                         \
-	   80,23   "file:../../sprites/shota0.png"   \
-	   150,23  "file:../../sprites/sgn2a0.png"   \
-	   210,18  "file:../../sprites/mguna0.png"   \
-	   10,38   "file:../../sprites/launa0.png"   \
-	   80,38   "file:../../sprites/plasa0.png"   \
-	   150,38  "file:../../sprites/bfuga0.png"   \
-	   240,38  "file:../../sprites/csawa0.png"   \
-	   10,67   "Ammo"                            \
-	   53,67   "file:../../sprites/clipa0.png"   \
-	   68,62   "file:../../sprites/ammoa0.png"   \
+	   20,38   "file:../../sprites/csawa0.png"   \
+	   90,23   "file:../../sprites/shota0.png"   \
+	   90,38   "file:../../sprites/sgn2a0.png"   \
+	   155,18  "file:../../sprites/mguna0.png"   \
+	   155,38  "file:../../sprites/launa0.png"   \
+	   235,23  "file:../../sprites/plasa0.png"   \
+	   235,38  "file:../../sprites/bfuga0.png"   \
+	   10,67   "Bullets"                         \
+	   77,62   "file:../../sprites/ammoa0.png"   \
```

Comment:
It seems sub-optimal having a this in a makefile with all the `\`s and whatnot. Like there could be a separate tab separated file with the coordinates followed by the text or file.

I know you didn't do it, so no need to change anything. Just my two cents on the way it currently works.

Reply:
Visually on my text editor it's fine, but if there's any way to do it that, say, lets you comment out one arbitrary line in the block that would be good...